cmd/cluster/health.go

```diff
@@ -177,15 +205,27 @@ func (o *healthOptions) run() error {
 	if err != nil {
 		log.Fatalf("error: %v", err)
 	}
+	fmt.Fprintf(o.IOStreams.Out, "\n \n")
```

Comment:
I don't think we need this.

Reply:
This line adds an empty line before writing the concise output, it makes the output look better IMO.  I get these Throttling request messages before the output comes, and without an empty line to separate the two, it wasn't easy to see where the command output starts.